src/main/java/svydovets/core/context/beanFactory/BeanFactory.java

```diff
@@ -112,21 +117,21 @@ private Object postProcessAfterInitialization(Object bean, String beanName) {
     private Object initWithBeanPostProcessor(String beanName, Object bean) {
         bean = postProcessBeforeInitialization(bean, beanName);
         postConstructInitialization(bean);
+
         return postProcessAfterInitialization(bean, beanName);
     }
 
-    private void postConstructInitialization(Object bean) {
+    private void postConstructInitialization(Object bean) throws NoUniquePostConstructException {
         Class<?> beanType = bean.getClass();
         Method[] declaredMethods = beanType.getDeclaredMethods();
         Predicate<Method> isAnnotatedMethod = method -> method.isAnnotationPresent(PostConstruct.class);
 
         // todo: Refactor a little bit. Collect methods to list to avoid second filtering
```

Comment:
Can we remove this `todo`?

Reply:
This should be fixed in another task. The scope of this task did not include